content/centralized.md

```diff
@@ -40,7 +40,7 @@ Features:
 - Typically all developers have both read and write permissions (double-headed arrows).
 - Suited for cases where **all developers are in the same group or organization or project**.
 - **Everybody who wants to contribute needs write access**.
-- Good idea to write-protect the main branch (typically `master` or `main`).
+- Good idea to write-protect the main branch (typically `main`).
```

Comment:
```suggestion
- Good idea to write-protect the main branch (typically `main` or `master`).
```

Reply:
In this case I would leave both. `master` is used in many repositories and it might take another decade until this changes.